.github/workflows/ci.yml

```diff
@@ -5,6 +5,23 @@ on:
   pull_request:
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v3
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install requirements
+      run: python -m pip install -r requirements/dev.txt
```

Comment:
Since you already have pip-tools managed in+txt file pair where the former is a direct requirements file and the latter is a lock file, you can use the proper semantics and make the intended invocation as follows:
```suggestion
      run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
```

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/6791bbf4a38719468006c4991bf5b12ef6cfb08d

Reply:
Marking this thread as resolved.